AirPal/Resources/CustomConfig.xcconfig

```diff
@@ -0,0 +1,12 @@
+//
+//  CustomConfig.xcconfig
+//  AirPal
+//
+//  Created by Marianna Mikhael on 4/18/24.
+//
+
+// Configuration settings file format documentation can be found at:
+// https://help.apple.com/xcode/#/dev745c5c974
+
+AVIATIONSTACK_API_KEY = KEY
```

Comment:
nit: might also wanna put the needed key in the README + quick instructions how to get the API key, in case future-you needs to redo these steps

Reply:
I love this suggestion! To avoid too much context-switching, I created this issue to track things that I might want to add to a README in the future #5